app/view_models/workarea/storefront/search_autocomplete_view_model.rb

```diff
@@ -21,8 +28,16 @@ def searches
         )
       end
 
-      def no_results?
-        products.empty? && searches.empty?
+      def trending_searches
+        @trending_searches ||= begin
+          Workarea::Insights::TrendingSearches.current.results.map do |v|
+            v['query_string']
+          end
+        end
+      end
+
+      def content
```

Comment:
i defined this in ContentSearch so you won't need it here.

Reply:
This is for outputting content blocks (see the autocomplete view). We'll need both.

Reply:
so should the method I defined be called `#contents` or something?